### ISSUE TITLE
Per-file comments do not allow trailing comments

### ISSUE THREAD

**Issue by bersbersbers** [microsoft/pyright#9237]
**Describe the bug**
In per-file comments, adding a trailing comment breaks the comment. This is not true for per-line comments.

**Code or Screenshots**
This works:

```python
{slice(0)}  # pyright: ignore[reportUnhashable]  # microsoft/pyright#9236
```

```
c:\Git\project\bug5.py
  c:\Git\project\bug5.py:1:1 - error: Expression value is unused (reportUnusedExpression)
1 error, 0 warnings, 0 informations
```

This does not:
```python
# pyright: reportUnhashable=false  # microsoft/pyright#9236
{slice(0)}
```

```
c:\Git\project\bug5.py
  c:\Git\project\bug5.py:1:29 - error: Pyright comment directive must be followed by "=" and a value of true, false, error, warning, information, or none
  c:\Git\project\bug5.py:2:1 - error: Expression value is unused (reportUnusedExpression)
  c:\Git\project\bug5.py:2:2 - error: Set entry must be hashable
    Type "slice" is not hashable (reportUnhashable)
3 errors, 0 warnings, 0 informations
```

While again, this does:

```python
# pyright: reportUnhashable=false
{slice(0)}
```

```
  c:\Git\project\bug5.py:2:1 - error: Expression value is unused (reportUnusedExpression)
1 error, 0 warnings, 0 informations
```

**VS Code extension or command-line**
`pyright 1.1.384`
